app/Http/Api/Schema/Wiki/StudioSchema.php

```diff
@@ -37,6 +37,7 @@ public function allowedIncludes(): array
     {
         return [
             AllowedInclude::make(AnimeSchema::class, Studio::RELATION_ANIME),
+            AllowedInclude::make(ExternalResourceSchema::class, Studio::RELATION_RESOURCES),
         ];
     }
 
```

Comment:
We need to add the StudioResource::ATTRIBUTE_AS field to the fields function. As it is right now, attempting to select this field for sparse fieldsets produces a validation error. 

Please use the same approach as AnimeSchema does to include the pivot field.